indico/htdocs/js/indico/jquery/trackrolewidget.js

```diff
@@ -0,0 +1,94 @@
+/* This file is part of Indico.
```

Comment:
wtforms widgets are in `htdocs/js/indico/widgets/` but so far we don't have anything else in there (i.e. no jquery ui widgets) so I think this one does belong into `indico/htdocs/js/indico/jquery/`